packages/elements/src/list/extensible-function.ts

```diff
@@ -1,7 +1,6 @@
 /**
- * Extensible function class
- * TODO: Move this to @refinitiv-ui/utils
- * ! Do not import this module !
+ * ExtensibleFunction is not CSP compliant. Do not use.
```

Comment:
My bad. Action should come first.
```suggestion
 *  Do not use: ExtensibleFunction is not CSP compliant.
```